lib/execution_engine2/utils/Condor.py

```diff
@@ -232,93 +123,70 @@ def _add_configurable_attributes(self, sub):
         return sub
 
     def _extract_resources_and_requirements(
-        self, sub: Dict[str, Any], cgrr: Dict[str, str]
-    ) -> Tuple[Dict[str, Any], str]:
+        self, sub: Dict[str, Any], job_reqs: JobRequirements
+    ) -> Dict[str, Any]:
         # Extract minimum condor resource requirements and client_group
-        resources = self.extract_resources(cgrr)
-        sub["request_cpus"] = resources.request_cpus
-        sub["request_memory"] = resources.request_memory
-        sub["request_disk"] = resources.request_disk
-        client_group = resources.client_group
+        sub["request_cpus"] = job_reqs.cpus
+        sub["request_memory"] = f"{job_reqs.memory_MB}MB"
+        sub["request_disk"] = f"{job_reqs.disk_GB}GB"
```

Comment:
See the discussion for the `ee2_scheduler_test.py` file for rationale for `MB` and `GB` here.